tests/csapi/ignored_users_test.go

```diff
@@ -0,0 +1,105 @@
+// +build !dendrite_blacklist
+
+// Rationale for being included in Dendrite's blacklist: https://github.com/matrix-org/dendrite/issues/600
+package csapi_tests
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/tidwall/gjson"
+
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/client"
+	"github.com/matrix-org/complement/internal/match"
+	"github.com/matrix-org/complement/internal/must"
+)
+
+// The Spec says here
+//     https://spec.matrix.org/v1.1/client-server-api/#server-behaviour-13
+// that
+// > Servers must not send room invites from ignored users to clients.
+//
+// Synapse does not have this property, as detailed in
+// https://github.com/matrix-org/synapse/issues/11506.
+// This reproduces that bug.
```

Comment:
Maybe?

```suggestion
// Synapse does not handle this properly, as detailed in
// https://github.com/matrix-org/synapse/issues/11506.
// This reproduces that bug.
```

I don't think this is true though since this is now fixed, it might be better to say:

> This is a regression test for `link to issue`.